redis/client.py

```diff
@@ -772,6 +779,7 @@ def __init__(self, host='localhost', port=6379,
                 'encoding_errors': encoding_errors,
                 'decode_responses': decode_responses,
                 'retry_on_timeout': retry_on_timeout,
+                'retry': copy.deepcopy(retry),
```

Comment:
nit: Since retry is added after client_name and username, I'd place it there in the DeprecationWarning.

Reply:
Not sure what you mean. What are we deprecating here.

Reply:
This was a result of GH folding. Ignore. Resolving.